third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 2.8.11)
+project(libvterm LANGUAGES C)
```

Comment:
If it is merged we can just install the libvterm msys package instead of building it ourselves. 

It would still be needed if building without msys2, but I'll take what i can get.